styles/src/index.ts

```diff
@@ -16,3 +16,35 @@ export function labels(source: string, key: string): LayerSpecification[] {
   const theme = themes[key];
   return labels_layers(source, theme);
 }
+
+export function layersWithCustomTheme(
+  source: string,
+  theme: Theme,
+): LayerSpecification[] {
+  return nolabels_layers(source, theme).concat(labels_layers(source, theme));
+}
+
+export function layersWithPartialCustomTheme(
+  source: string,
+  key: string,
+  partialTheme: Partial<Theme>,
+): LayerSpecification[] {
+  const mergedTheme = { ...themes[key], ...partialTheme };
+  return nolabels_layers(source, mergedTheme).concat(
+    labels_layers(source, mergedTheme),
+  );
+}
+
+export function noLabelsWithCustomTheme(
```

Comment:
I would prefer not to have these two functions that just call another function with the same arguments.

Reply:
I'm going to merge this for now, we can think about a cleaner API in styles 4.0 once we understand the use cases better :P